README.md

```diff
@@ -6,6 +6,8 @@ This repo contains the sources for the [OpenShift.org](https://www.openshift.org
 
 ## Development
 
+**Note** (`Node.js` is a required dependency)[http://stackoverflow.com/a/6283074/6758654] on Linux machines
```

Comment:
Good gawd, I have such a mental block in this aspect...I *always* switch them! Thanks for pointing that out =)